dec/decode.c

```diff
@@ -784,9 +784,9 @@ BrotliResult WriteRingBuffer(BrotliOutput output,
   return BROTLI_RESULT_SUCCESS;
 }
 
-BrotliResult BROTLI_NOINLINE CopyUncompressedBlockToOutput(BrotliOutput output,
-                                                           int pos,
-                                                           BrotliState* s) {
+static BrotliResult BROTLI_NOINLINE CopyUncompressedBlockToOutput(BrotliOutput output,
```

Comment:
I just noticed that this goes beyond the 80-character line limit that is observed in this project. Could you please break the argument list to the next line? Sorry for the nitpicking.